packages/expo-in-app-purchases/src/InAppPurchases.ts

```diff
@@ -155,19 +157,25 @@ export async function getPurchaseHistoryAsync(
  * service on iOS.
  *
  * @param itemId The product ID of the item you want to buy.
- * @param oldPurchaseToken __Android Only.__ The `purchaseToken` of the purchase that the user is
- * upgrading or downgrading from. This is mandatory for replacing an old subscription such as when
- * a user upgrades from a monthly subscription to a yearly one that provides the same content.
- * You can get the purchase token from [`getPurchaseHistoryAsync`](#inapppurchasesgetpurchasehistoryasyncrefresh-boolean).
+ * @param details __Android Only.__ details for billing flow, can be one of:
+ *  - 'oldPurchaseToken' : the `purchaseToken` of the purchase that the user is upgrading or downgrading from. This is mandatory
+ *    for replacing an old subscription such as when a user upgrades from a monthly subscription to a yearly
+ *    one that provides the same content. You can get the purchase token from [`getPurchaseHistoryAsync`](#inapppurchasesgetpurchasehistoryasyncrefresh-boolean).
+ *  - 'obfuscatedAccountId' : the obfuscated account id of the user's Google Play account.
+ *  - 'obfuscatedProfileId' : the obfuscated profile id of the user's Google Play account.
```

Comment:
I think this should be:

"the obfuscated [account|profile] ID of the user in the app's system."

Google knows the account of the user - the problem is, we don't get that info. So this is _our_ ID, and Google wants it in an obfuscated form.

Reply:
Done :)